src/modules/events/components/EventListNavigation.tsx

```diff
@@ -34,13 +38,22 @@ export const EventListNavigation = observer(() => {
 
 	return (
 		<div className='events-nav'>
-			<button className='button-base' onClick={getPrevEvents}>
+			<Chip
+				className='events-nav__arrow-wrapper events-nav__arrow-wrapper--left'
```

Comment:
we actually don't use modifiers like that. this way you should repeat whole class name twice
instead just declare a class .left
and classname will be `events-nav__arrow-wrapper left`


also wrapper is a toxic word for css classes i think, once you start doing that everything becomes a wrapper

Reply:
should i change it to events-nav__arrow?

Reply:
events-nav__arrow is ok